VRDR/IJEMortality.cs

```diff
@@ -937,7 +937,8 @@ public string AUXNO
             {
                 if (!String.IsNullOrWhiteSpace(value))
                 {
-                    LeftJustified_Set("AUXNO", "StateLocalIdentifier1", value);
+                    value = value.PadLeft(12 , '0');
+					LeftJustified_Set("AUXNO", "StateLocalIdentifier1", value);
```

Comment:
Please fix the spacing, set your editor to convert tabs to spaces so it's consistent.